data/json/bionics.json

```diff
@@ -415,18 +676,30 @@
         "type": "bionic",
         "id": "bio_evap",
         "name": "Aero-Evaporator",
+        "occupied_bodyparts": [
+            [ "TORSO", 5 ],
+            [ "HAND_R", 1 ]
+        ],
         "act_cost": 200,
         "description": "This unit draws moisture from the surrounding air, which then is poured from a fingertip in the form of water.  It may fail in very dry environments."
     },{
         "type": "bionic",
         "id": "bio_lighter",
         "name": "Mini-Flamethrower",
+        "occupied_bodyparts": [
+            [ "HAND_L", 1 ],
+            [ "HAND_R", 1 ]
+        ],
         "act_cost": 75,
         "description": "The index fingers of both hands have powerful fire starters which extend from the tip."
```

Comment:
I just realized how stupid this bionic is. Index-finger sized flamethrower creating raging fires in no time? Using what to create the fire?

Would make sense to move it to the arm instead.